compiler/test/fail_compilation/ice18753.d

```diff
@@ -3,7 +3,7 @@ TEST_OUTPUT:
 ---
 fail_compilation/ice18753.d(21): Error: variable `ice18753.isInputRange!(Group).isInputRange` - type `void` is inferred from initializer `ReturnType(func...)`, and variables cannot be of type `void`
 fail_compilation/ice18753.d(23): Error: template instance `ice18753.isInputRange!(Group)` error instantiating
-fail_compilation/ice18753.d(18):        instantiated from here: `isForwardRange!(Group)`
+fail_compilation/ice18753.d(23):        instantiated from: `isForwardRange!(Group)` at fail_compilation/ice17853.d(18)
 fail_compilation/ice18753.d(18):        while evaluating: `static assert(isForwardRange!(Group))`
```

Comment:
Embedding file/line in the message instead of cross referencing is not an improvement. Instantiation backtraces would suffer the most for this when vcontext is enabled.